showcase/src/assets/examples/table/with-items-per-page-selector/table.ts

```diff
@@ -0,0 +1,90 @@
+import { Component, Inject } from "@angular/core";
+import { STARK_LOGGING_SERVICE, StarkLoggingService } from "@nationalbankbelgium/stark-core";
+import { StarkPaginationConfig, StarkTableColumnProperties, StarkTableFilter, StarkTableRowActions } from "@nationalbankbelgium/stark-ui";
+
+const DUMMY_DATA: object[] = [
```

Comment:
Could you please reduce this object as we did in another example: https://github.com/NationalBankBelgium/stark/blob/52ae6e0380e24bd3678f0df15edd53746c6ac79c/showcase/src/assets/examples/table/with-custom-styling/table.ts#L4 

```typescript
const DUMMY_DATA: object[] = [
	{ id: 1, title: { label: "first title (value: 1)", value: 1 }, description: "number one" },
	/* ... */
	{ id: 44, title: { label: "twelfth title (value: 12)", value: 12 }, description: "the twelfth description" }
];
```